lib/common/patchset.c

```diff
@@ -1080,17 +1092,18 @@ xml_apply_patchset(xmlNode *xml, xmlNode *patchset, bool check_version)
             default:
                 crm_err("Unknown patch format: %d", format);
                 rc = -EINVAL;
+                break;
         }
     }
 
     if ((rc == pcmk_ok) && (digest != NULL)) {
-        char *new_digest = NULL;
-        char *version = crm_element_value_copy(xml, XML_ATTR_CRM_VERSION);
+        const char *version = crm_element_value_copy(xml, XML_ATTR_CRM_VERSION);
```

Comment:
I suppose you meant to change this to crm_element_value()?

Reply:
Yep, I'm pretty sure I did at one point and then messed it up in a merge conflict resolution.